js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
Maybe like this
![f113f23e-37d7-11e6-8dc8-3de13731873f](https://cloud.githubusercontent.com/assets/1374172/16243119/0c6b8cb8-37f6-11e6-8fda-4e3d356a181d.png)
cc @skjnldsv @jancborchardt @Henni